src/ytdl_sub/prebuilt_presets/tv_show/new_w_enable.yaml

```diff
@@ -0,0 +1,185 @@
+presets:
+
+####################################################################################################
+
+  "Plex":
+    overrides:
+      player: "plex"
+
+  "Jellyfin":
+    overrides:
+      player: "jellyfin"
+
+  "Kodi":
+    overrides:
+      player: "kodi"
+
+  # "TV Show":
+  #   output_options:
+  #     output_directory: "{tv_show_directory}/{tv_show_name_sanitized}"
+  #     file_name: "{episode_file_path}.{ext}"
+  #     thumbnail_name: "{thumbnail_file_name}"
+  #     info_json_name: "{episode_file_path}.{info_json_ext}"
+  #     maintain_download_archive: True
+
+  #   ytdl_options:
+  #     break_on_existing: True
+
+  #   chapters:
+  #     embed_chapters: True
+
+  #   video_tags:
+  #     show: "{tv_show_name}"
+  #     genre: "{tv_show_genre}"
+  #     episode_id: "{episode_number}"
+  #     title: "{episode_title}"
+  #     synopsis: "{episode_plot}"
+  #     year: "{episode_year}"
+  #     date: "{episode_date_standardized}"
+  #     contentRating: "{episode_content_rating}"
+
+  #   nfo_tags:
+  #     enable: "{nfo_tags_enable}"
+  #     kodi_safe: "{kodi_safe_enable}"
+  #     nfo_name: "{episode_file_path}.nfo"
+  #     nfo_root: "episodedetails"
+  #     tags:
+  #       genre:
+  #        - "{tv_show_genre}"
+  #       season: "{season_number}"
+  #       episode: "{episode_number}"
+  #       title: "{episode_title}"
+  #       plot: "{episode_plot}"
+  #       year: "{episode_year}"
+  #       aired: "{episode_date_standardized}"
+  #       mpaa: "{episode_content_rating}"
+
+  #   output_directory_nfo_tags:
+  #     enable: "{nfo_tags_enable}"
+  #     kodi_safe: "{kodi_safe_enable}"
+  #     nfo_name: "tvshow.nfo"
+  #     nfo_root: "tvshow"
+  #     tags:
+  #       title: "{tv_show_name}"
+  #       genre:
+  #        - "{tv_show_genre}"
+  #       mpaa: "{tv_show_content_rating}"
+
+  #   file_convert:
+  #     enable: "{file_convert_enable}"
+  #     convert_to: "mp4" 
+
+  #   format: "{format_selector}"
+
+  #   overrides:
+  #     #   MUST DEFINE:
+  #     #     tv_show_directory
+  #     #     season_number
+  #     #     season_number_padded
+  #     #     episode_number
+  #     #     episode_number_padded
+  #     file_uid_default: "{uid_sanitized}"
+  #     file_title_default: "{title_sanitized}"
+  #     file_uid_plex: "{uid_sanitized_plex}"
+  #     file_title_plex: "{title_sanitized_plex}"
+
+
+  #     file_uid: >-
+  #       {
+  #         %if(
+  #           %contains(player, 'plex'),
+  #           file_uid_plex,
+  #           file_uid_default
+  #         )
+  #       }
+  #     file_title: >-
+  #       {
+  #         %if(
+  #           %contains(player, 'plex'),
+  #           file_title_plex,
+  #           file_title_default
```

Comment:
when the time comes in a follow-up PR, we can use the values directly instead defining extra variables, i.e.
```
  #     file_uid: >-
  #       {
  #         %if(
  #           %contains(player, 'plex'),
  #           uid_sanitized,
  #           uid_sanitized_plex
  #         )
  #       }
  ```